### PR TITLE
MPL circuit drawer left align when the circuit is short

### LINKED ISSUE #2788: Gates are not left justified in mpl circuit drawer
### Information

- **Qiskit Terra version**: master
- **Python version**:
- **Operating system**:

### What is the current behavior?
There is always a space on the line between the qubit label and the first gate:
circ_color

To reduce the width of the figure as much as possible, the gates should be drawn as far left as possible.

### PR DESCRIPTION
**Pull request by abhobe**
### Summary
-  Removed beginning fold lines.
 - Fixes #2788 by @nonhermitian 
Binder Tests: [Binder](https://mybinder.org/v2/gh/abhobe/qiskit-terra/2788)

### Details and comments
Much of the credit goes to @enavarro51 for helping me on this.

### CODE REVIEW

**Comment by abhobe** [comment]
@1ucian0 Binder Tests Are Ready

File: qiskit/visualization/matplotlib.py [hunk 1]
```diff
@@ -699,28 +689,33 @@ def _draw_regs_sub(self, n_fold, feedline_l=False, feedline_r=False):
...
+                self.ax.plot([xpos_l, xpos_l], [ypos1, ypos2],
+                             color=self._style.lc, linewidth=self._lwidth15, zorder=PORDER_LINE)
+            if feedline_r:
+                self.ax.plot([xpos_r, xpos_r], [ypos1, ypos2],
+                             color=self._style.lc, linewidth=self._lwidth15, zorder=PORDER_LINE)
 
     def _draw_ops(self, verbose=False):
-        _standard_gates = ['x', 'y', 'z', 'id', 'h', 'r', 's', 'sdg', 't', 'tdg', 'rx', 'ry', 'rz',
-                           'rxx', 'ryy', 'rzx', 'u1', 'u2', 'u3', 'swap', 'reset', 'sx', 'sxdg']
+        _standard_1q_gates = ['x', 'y', 'z', 'id', 'h', 'p', 'r', 's', 'sdg', 't', 'tdg', 'rx',
```
Comment (1ucian0) [inline qiskit/visualization/matplotlib.py:717]: what's `p`?

Comment (abhobe) [reply qiskit/visualization/matplotlib.py:717]: It was a test gate let me remove it

Comment (abhobe) [reply qiskit/visualization/matplotlib.py]: Removed it

**Comment by 1ucian0** [comment]
All the references were changed (see https://mybinder.org/v2/gh/abhobe/qiskit-terra/2788?urlpath=apps/test/ipynb/mpl_tester.ipynb) so updating in b86e7f4f3

**Review by 1ucian0 — APPROVED** [review]
Thanks!

**Comment by abhobe** [comment]
Will their need to be updates on documentation to match the new style of mpl?

**Comment by 1ucian0** [comment]
> Will their need to be updates on documentation to match the new style of mpl?

It is generated independently. It will be updated in the next cycle.

**Comment by abhobe** [comment]
Oh ok, I will see if I can help with that.

File: qiskit/visualization/matplotlib.py (hunk 1 above)
Comment (1ucian0) [reply qiskit/visualization/matplotlib.py]: hold on.. it's the PhaseGate...

**Review by 1ucian0 — APPROVED** [review]

**Pull request merged**